src/components/Alert.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { shallowEqual } from 'react-redux'
+import { useSelector } from 'react-redux'
+
+
+
+const Alert = () => {
+
+
+    const { alerts } = useSelector(state => ({
+        alerts : state.alerts
+    }), shallowEqual)
+
+    return(
+        <>
+            {
+                alerts.map(alert => <div key={alert.id}>[{alert.alertType}] : {alert.msg}</div>)
```

Comment:
Why not just use the `alert.alertType` to set the background color once and for all.

Reply:
That's UI stuff... we aren't focusing on UI for now... Just functionality